src/jane/JaneBurner.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.22;
+
+import {JaneToken} from "./JaneToken.sol";
+
+contract JaneBurner {
+    error NotOwner();
+    error Unauthorized();
+
+    event AuthorizationUpdated(address indexed account, bool status);
+
+    JaneToken public immutable JANE;
+
+    mapping(address => bool) public authorized;
+
+    constructor(address jane) {
```

Comment:
I don't like this two-sided dependency. 

plus this is not an upgradeable contract which defeats the purpose of having a secondary contract

at that point might as well move this logic over to JANE itself and have access control for it.